### PR TITLE
Update Net docs

### PR DESCRIPTION
**Pull request by danmoseley**
Literal translation from the code xml docs using porting tool

Contributes to https://github.com/dotnet/runtime/issues/54857

### CODE REVIEW

**Comment by ManickaP** [comment]
This still contains QUIC APIs, from which preview version is this generated? I'm asking whether we should remove the docs manually now or keep waiting until it gets removed automagically.

**Comment by danmoseley** [comment]
@ManickaP I generated from head about a week ago. do you expect more changes to the public surface since then? If so I can run it again.

note that the "slots" that the text goes into are generated by the docs build process reflecting over the assemblies. I do not know how often they do this; that would create docs for the Quic API if it's present in what they scan. If the docs have "slots" I am not sure whether the porting tool will bring over text if the API has since become non-public.

maybe it's easiest to just commit this, and then snip all the Quic API in a followup PR

**Comment by ManickaP** [comment]
My primary concern is to prevent publishing S.N.Quic docs, how we achieve that doesn't matter. 
I really don't see inside the tooling. I don't know whether it'll trim it on it's own, once they see the API became private or whether we need to do that manually.
I'm just raising awareness of this everywhere S.N.Quic docs is touched.

And yes, we can merge this now and do the clean up in a follow up PR.

**Review by ManickaP — APPROVED** [review]

**Comment by carlossanlop** [comment]
@safern can we send the latest assemblies to @gewarren to ensure we submit the correct documentation to MS Docs?

@ManickaP if the Quic APIs are expected to go away because they are now private, then once we update the assemblies in the Docs server, you won't have to do anything.

File: xml/System.Net.Http.Headers/HeaderStringValues.xml [hunk 1]
```diff
@@ -150,7 +158,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a string containing all the headers in the collection.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Net.Http.Headers/HeaderStringValues.xml:162]:
```suggestion
        <returns>The string representation of all the headers in the collection.</returns>
```

File: xml/System.Net.Http.Headers/HeaderStringValues.xml [hunk 2]
```diff
@@ -78,7 +78,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets an enumerator for all of the strings in the collection.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Net.Http.Headers/HeaderStringValues.xml:82]:
```suggestion
        <returns>An enumerator for all the strings in the collection.</returns>
```

File: xml/System.Net.Http.Headers/HttpHeadersNonValidated.xml [hunk 3]
```diff
@@ -268,7 +290,7 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Net.Http.Headers.HeaderStringValues&gt;</ReturnType>
```
Comment (carlossanlop) [inline xml/System.Net.Http.Headers/HttpHeadersNonValidated.xml:290]: FYI there's an undoc API in lines 267-270 of this file.

File: xml/System.Net.Sockets/Socket.xml [hunk 4]
```diff
@@ -11379,11 +11397,15 @@ This member outputs trace information when you enable network tracing in your ap
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="fileName">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="fileName">A <see cref="T:System.String" /> that contains the path and name of the file to be sent. This parameter can be <see langword="null" />.</param>
+        <param name="cancellationToken">A cancellation token that can be used to cancel the asynchronous operation.</param>
+        <summary>Sends the file <paramref name="fileName" /> to a connected <see cref="T:System.Net.Sockets.Socket" /> object.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Net.Sockets/Socket.xml:11403]:
```suggestion
        <returns>A value task that represents the asynchronous send file operation.</returns>
```

File: xml/System.Net.Sockets/Socket.xml [hunk 5]
```diff
@@ -11417,14 +11439,18 @@ This member outputs trace information when you enable network tracing in your ap
...
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="fileName">A <see cref="T:System.String" /> that contains the path and name of the file to be sent. This parameter can be <see langword="null" />.</param>
+        <param name="preBuffer">A <see cref="T:System.Byte" /> array that contains data to be sent before the file is sent. This parameter can be <see langword="null" />.</param>
+        <param name="postBuffer">A <see cref="T:System.Byte" /> array that contains data to be sent after the file is sent. This parameter can be <see langword="null" />.</param>
+        <param name="flags">One or more of <see cref="T:System.Net.Sockets.TransmitFileOptions" /> values.</param>
+        <param name="cancellationToken">A cancellation token that can be used to cancel the asynchronous operation.</param>
+        <summary>Sends the file <paramref name="fileName" /> and buffers of data to a connected <see cref="T:System.Net.Sockets.Socket" /> object using the specified <see cref="T:System.Net.Sockets.TransmitFileOptions" /> value.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Net.Sockets/Socket.xml:11448]:
```suggestion
        <returns>A value task that represents the asynchronous send file operation.</returns>
```

File: xml/System.Net.WebSockets/WebSocket.xml [hunk 6]
```diff
@@ -435,9 +435,9 @@
         <Parameter Name="options" Type="System.Net.WebSockets.WebSocketCreationOptions" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="stream">The <see cref="T:System.IO.Stream" /> for the connection.</param>
+        <param name="options">The options with which the websocket must be created.</param>
+        <summary>Creates a <see cref="T:System.Net.WebSockets.WebSocket" /> that operates on a <see cref="T:System.IO.Stream" /> representing a web socket connection.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Net.WebSockets/WebSocket.xml:441]:
```suggestion
        <returns>The created web socket.</returns>
```

**Review by carlossanlop — APPROVED** [review]
LGTM. I added a few suggestions which I'm going to apply.

**Pull request merged**